languages/php/INSTALL.md

```diff
@@ -0,0 +1,53 @@
+# PHP Installation
+
+## Introduction
+
+Composer is used to build PHP Bitwarden client library.
+
+## Prerequisites
+
+- PHP >= 8.0
+- FFI extension enabled in PHP configuration
+- Composer
+- Bitwarden SDK native library, which is expected in one of below locations, depending on the OS and architecture.
+  If you prefer to build SDK yourself, see [SDK README.md](../../README.md) for instructions.
+    - Windows x86_64: `/lib/windows-x64/bitwarden_c.dll`
+    - Linux x86_64: `/lib/linux-x64/libbitwarden_c.so`
+    - macOS x86_64: `/lib/macos-x64/libbitwarden_c.dylib`
+    - macOS aarch64: `/lib/macos-arm64/libbitwarden_c.dylib`
```

Comment:
I might be misunderstanding this, or I lack the knowledge on how PHP stuff works.

Is this from the root of the file system? 

I don't believe macOS will let you make `/lib/`. 

This also seems weird for Windows.

Couple with the changes in `languages/php/src/BitwardenLib.php` could the original intent be to have a `lib` directory in project's working directory?

Reply:
You are right, i did not know you can't have custom directories in macOS. I will revert this change.
Also i did test it on windows and the `/` is equivalent to current workspace drive letter, usually `C:\`. I will add this to the documentation.